diff --git a/packages/backend/ormconfig.js b/packages/backend/ormconfig.js
index 229e5bf1f..9650493eb 100644
--- a/packages/backend/ormconfig.js
+++ b/packages/backend/ormconfig.js
@@ -6,12 +6,12 @@ const config = loadConfig();
 
 export default new DataSource({
 	type: 'postgres',
-	host: config.db.host,
-	port: config.db.port,
-	username: config.db.user,
-	password: config.db.pass,
-	database: config.db.db,
-	extra: config.db.extra,
+	host: config.db?.host,
+	port: config.db?.port,
+	username: config.db?.user,
+	password: config.db?.pass,
+	database: config.db?.db,
+	extra: config.db?.extra,
 	entities: entities,
 	migrations: ['migration/*.js'],
 });

